app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+    skip_before_action :verify_authenticity_token, only: [:create]
+  
+    def create
+      user = User1.find_by(username: params[:username])
```

Comment:
Why model is named as `User1`?

Reply:
Actually before jwt I authenticated with devise so user table was created in db, and when i created User table for jwt its giving error that User table already exist.

Reply:
Why create separate table for JWT?
You can use the earlier one, right?